problems/euler/52/52.py

```diff
@@ -0,0 +1,39 @@
+"""
+Euler Problem: 52
+
+It can be seen that the number, 125874, and its double, 251748, 
+contain exactly the same digits, but in a different order.
+
+Find the smallest positive integer, x, such that 
+2x, 3x, 4x, 5x, and 6x, contain the same digits.
+
+"""
+
+def permutedMutables(num):
+	dig_set = set(str(num))
+
+	for i in range(2, 7):
+		val = num * i
+		val_set = set(str(val))
+
+		if not dig_set == val_set:
+			return False
+
+	return True
+
+def euler52():
+
```

Comment:
can we remove these empty lines before merging?

Reply:
I have removed the empty lines.